### PR TITLE
Support glob in sources

### PR DESCRIPTION
**Pull request by dobladov**
Fixes: #25

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,7 +28,8 @@
   "homepage": "https://github.com/zattoo/changelog#readme",
   "dependencies": {
     "@actions/core": "1.2.4",
-    "@actions/github": "4.0.0"
+    "@actions/github": "4.0.0",
+    "glob-promise": "3.4.0"
```
Comment (godban) [inline package.json:32]: why not https://github.com/isaacs/node-glob ?

Comment (dobladov) [reply package.json:32]: This one uses glob internally, with normal glob you have to use callbacks this is just a wrapper to use promises.

Comment (godban) [reply package.json:32]: using origin lib with callback is totally fine, depending on proxy lib for syntax sugar is so so

Comment (dobladov) [reply package.json]: Refactored now to use node-glob and `util.promisify`.

**Pull request merged**